example/mqttapp/mqttapp.mk

```diff
@@ -8,7 +8,7 @@ $(NAME)_SOURCES     := mqtt-example.c
 
 $(NAME)_COMPONENTS := cli  protocol.linkkit.iotkit  connectivity.mqtt cjson fota netmgr framework.common 
 
-LWIP := 0
```

Comment:
This file has the same problem as file mqtt-example.c. It will affect other board for this common file.